frontend/next.config.ts

```diff
@@ -21,8 +21,8 @@ const defaultBackendConfig: BackendConfig = {
 
 const config = merge(
   {
-    backend: defaultBackendConfig,
-    backendSsr: defaultBackendConfig,
+    backend: cloneDeep(defaultBackendConfig),
```

Comment:
I found out we actually use external backend configuration on production.
Fetching can get a little faster with proper separation of backend configs.

Reply:
Moved it to a separate commit: https://github.com/near/near-explorer/commit/1aa73af0bc310611aab1b41337c825eb02e8ce6c